test/bdd/fixtures/user-agent/docker-compose.yml

```diff
@@ -438,8 +438,8 @@ services:
       - KMS_TLS_SERVE_KEY=/etc/tls/ec-key.pem
       - KMS_DATABASE_TYPE=mem
       - KMS_SECRETS_DATABASE_TYPE=mem
-      - KMS_OPERATIONAL_KMS_STORAGE_TYPE=edv
-      - KMS_OPERATIONAL_KMS_STORAGE_URL=https://edv.example.com:${OPS_EDV_PORT}
+      - KMS_OPERATIONAL_KMS_STORAGE_TYPE=sds
```

Comment:
I'm guessing that the KeyServer still uses the "sds" name? Can we push the name change there instead?
(I can take care of this)

Reply:
I'll update once KMS is updated. For now, its using `sds`.

Reply:
Created issue for it: https://github.com/trustbloc/hub-kms/issues/88

(I'll take care of this in the future)